src/main/java/htsjdk/samtools/SAMFileWriterFactory.java

```diff
@@ -550,9 +550,11 @@ public CRAMFileWriter makeCRAMWriter(final SAMFileHeader header, final File outp
      * @param referenceFasta reference sequence file
      * @return CRAMFileWriter
      *
+     * @deprecated since 6/18, use {@link #makeCRAMWriter(SAMFileHeader, boolean, Path, Path)} instead
```

Comment:
@lbergelson Hm I dropped my connection. Anyway, should we suggest migration to `makeWriter(SAMFileHeader, boolean, Path, Path)` as the canonical all-knowing method. And someday deprecate all the rest ?

Reply:
Oh, you're right.  I'll change.